### PR TITLE
Include all products in the supported versions list

### LINKED ISSUE #121: Document supported product versions
I'd like a page that summarizes for each operator the supported product versions.

Please see the epic at https://github.com/stackabletech/issues/issues/139 for more details.
Ideally the information can be aggregated from the operators themselves so the content is only written once.

### PR DESCRIPTION
**Pull request by teozkr**
Fixes #121

### CODE REVIEW

File: modules/ROOT/pages/supported-versions.adoc [hunk 1]
```diff
@@ -2,10 +2,46 @@
...
-include::zookeeper::partial$supported-versions.adoc[]
+include::airflow::partial$supported-versions.adoc[]
+
+== Apache Druid
+
+include::druid::partial$supported-versions.adoc[]
+
+== Apache HBase
+
+include::hbase::partial$supported-versions.adoc[]
```
Comment (teozkr) [inline modules/ROOT/pages/supported-versions.adoc:15]: This depends on https://github.com/stackabletech/hbase-operator/commit/974c6e99aec068e47293ab83ae074674e8769f41 (from the rework branch), and will cause an ugly warning on the docs page until then.

That's not great, but it still seems like a better outcome long-term then forgetting to include HBase once the rework is merged.

**Review by maltesander — APPROVED** [review]
LGTM. Do we still want to wait for HBase to avoid that warning so far?

**Comment by teozkr** [comment]
@lfrancke @siegfriedweber What do you think?

File: modules/ROOT/pages/supported-versions.adoc (hunk 1 above)
Comment (lfrancke) [reply modules/ROOT/pages/supported-versions.adoc:15]: Agreed!

**Review by lfrancke — APPROVED** [review]

**Pull request merged**